cypress/integration/demographQuestions.spec.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable scanjs-rules/identifier_localStorage, max-len */
```

Comment:
Noting here I disable `max-len`. I think enforcing this on tests makes them less readable because of the text descriptions.